documentation/user/Information center.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: wiki
+title: Information center
+description: Downloaded ACE3 and have no idea where to start? This page serves as a document to help new players get started with things or get an answer to some of your questions.
+group: user
+order: 0
+parent: wiki
+---
+
+Downloaded ACE3 and have no idea where to start? This page serves as a document to help new players and mission makers understand what's available to them.
+
+
+- You don't know where to begin your ACE3 journey? [**Check out ACE3 features**](http://ace3mod.com/wiki/feature/)
+
+
+- You are a mission maker but you don't know what ACE3 have to offer? [**We have some documentation for you**](http://ace3mod.com/wiki/missionmaker/)
+
+
+- Are you searching for ACE3 classnames ? [**Here they are**](http://ace3mod.com/wiki/missionmaker/classnames.html)
+
+
+## 1. FAQ
+### 1.1 Features 
+<b>Q:</b> Where is X feature? </br>
+<b>A:</b> When it's done.</br>
```

Comment:
Shouldn't answer with `when` on a `where` question.
Maybe `When will we see feature x?`